addons/pos_loyalty/static/tests/tours/pos_loyalty_reward_button_tour.js

```diff
@@ -142,6 +141,20 @@ registry.category("web_tour.tours").add("PosLoyaltyFreeProductTour2", {
         ].flat(),
 });
 
+registry.category("web_tour.tours").add("PosLoyaltyFreeProductTour3", {
```

Comment:
We now try to give the same name to the test and tours so that we can find them easily
```suggestion
registry.category("web_tour.tours").add("test_loyalty_free_product_rewards_2", {
```